mediator/src/controllers/patient.js

```diff
@@ -0,0 +1,46 @@
+const axios = require('axios');
+const FHIR_ROOT_URL = 'http://openhim-core:5001/fhir';
+
+function genereateFHITPatientResource(patient) {
+  const patientLastName = patient.name.split(' ').slice(-1);
+  const FHITPatientResource = {
+    resourceType: 'Patient',
+    id: patient.id,
+    identifier: [
+      {
+        system: 'cht',
+        value: patient._id
+      }
+    ],
+    name: [
+      {
+        use: 'official',
+        family: patientLastName,
+        given: [patient.name]
+      }
+    ],
+    gender: patient.sex,
+    birthDate: patient.date_of_birth
+  };
+
+  return FHITPatientResource;
+}
+
+async function createPatient(req) {
```

Comment:
We only need `req.body` to create the patient, we'll have to isolate the controllers from the routing library.